### PR TITLE
Integrate Schedule Publish

### PR DESCRIPTION
**Pull request by graymd**
JIRA: https://agile.lootcrate.com/browse/SCE-28

This PR utilizes sidekiq to add schedule publish feature to Alchemy.

Corresponding LC PR:  https://github.com/LootCrate/lootcrate/pull/3239

### CODE REVIEW

File: app/controllers/alchemy/admin/pages_controller.rb [hunk 1]
```diff
@@ -39,6 +39,43 @@ def info
         render layout: !request.xhr?
       end
 
+      def schedule_publish_new
+        render layout: !request.xhr?
+      end
+
+      def schedule_publish
+        delete_current_scheduled_job(@page.job_id) if @page.job_id
+        jobid = ScheduleCmsPagePublish.perform_at(time_to_publish_utc, @page.id)
```
Comment (jsqu99) [inline app/controllers/alchemy/admin/pages_controller.rb]: nitpicking but why not job_id ?

**Comment by jsqu99** [comment]
LGTM

File: app/controllers/alchemy/admin/pages_controller.rb (hunk 1 above)
Comment (graymd) [inline app/controllers/alchemy/admin/pages_controller.rb]: updated.

File: app/controllers/alchemy/admin/pages_controller.rb [hunk 2]
```diff
@@ -39,6 +39,43 @@ def info
...
+        redirect_to admin_pages_path
+      end
+
+      def time_to_publish_pst
+        year = params[:scheduled_publish]['time_to_publish(1i)'].to_i
+        month = params[:scheduled_publish]['time_to_publish(2i)'].to_i
+        day = params[:scheduled_publish]['time_to_publish(3i)'].to_i
+        hours = params[:scheduled_publish]['time_to_publish(4i)'].to_i
+        minutes = params[:scheduled_publish]['time_to_publish(5i)'].to_i
+        @time_to_publish_pst ||= Time.zone.parse("#{year}-#{month}-#{day} #{hours}:#{minutes}")
```
Comment (AlanLattimore) [inline app/controllers/alchemy/admin/pages_controller.rb:60]: Do you want to be explicit about the timezone here? For critical calculations (like 'go live at midnight') Is it enough to rely on the system default which might change and throw things out of whack? (I just don't know.)

I know the calculations for rebill dates are all based on ET/EST not PT/PST. You've specified 'All times in Pacific' so I think you're good here.

**Comment by AlanLattimore** [comment]
LGTM

**Pull request merged**